Gemfile.lock

```diff
@@ -912,4 +910,4 @@ RUBY VERSION
    ruby 3.0.3p157
 
 BUNDLED WITH
-   2.4.3
+   2.4.4
```

Comment:
You got the wrong bundler installed! :wink: 

We better update this in a separate PR.